server/src/routes/api/v1/users/users-v1.route.ts

```diff
@@ -1,13 +1,13 @@
 import { Router, Request, Response, NextFunction } from "express";
```

Comment:
I wouldn't use the v1 version indicator so much because the version is already visible from the directory tree (v1 directory). I think thats redundant and makes the name harder to read